util.py

```diff
@@ -0,0 +1,34 @@
+def mean(xs):
+    """Mean of an iterable"""
+    sum = 0
+    count = 0
+    for x in xs:
+        sum += x
+        count += 1
+    return sum / count
+
+def show(n):
```

Comment:
there's probably a format directive for this

Reply:
Yup, apparently you can do `"{:+}".format(3) == "+3"`.